src/main/java/bio/terra/common/FlightUtils.java

```diff
@@ -60,4 +62,12 @@ public static boolean isBigQueryIamPropagationError(BigQueryException ex) {
         }
         return false;
     }
+
+    public static RetryRuleRandomBackoff getDefaultRandomBackoffRetryRule(final int maxConcurrency) {
```

Comment:
Could this not be done inside `RetryRuleRandomBackoff`? That seems like a more natural place to define an alternate default constructor for this object.

Reply:
The thing is that the retry rules are part of a separate repo (the stairway repo) and these defaults are application specific so that's why I put them here